apps/heft/src/plugins/JestPlugin/JestPlugin.ts

```diff
@@ -145,6 +141,45 @@ export class JestPlugin implements IHeftPlugin {
     clean.properties.pathsToDelete.add(cacheFolder);
   }
 
+  private _getJestReporters(
+    heftSession: HeftSession,
+    heftConfiguration: HeftConfiguration
+  ): (string | Config.ReporterConfig)[] {
+    const config: Config.GlobalConfig = JsonFile.load(this._getJestConfigFolder(heftConfiguration));
+
+    if (Array.isArray(config.reporters)) {
+      // Replace the 'default' reporter with the heft reporter
+      const defaultIndex: number = config.reporters.indexOf('default');
+      if (defaultIndex >= 0) {
+        config.reporters[defaultIndex] = this._getHeftJestReporterConfig(heftSession, heftConfiguration);
+      }
+
+      return config.reporters;
+    } else {
+      // Otherwise if no reporters are specified install only the heft reporter
+      return [this._getHeftJestReporterConfig(heftSession, heftConfiguration)];
+    }
+  }
+
+  private _getHeftJestReporterConfig(
+    heftSession: HeftSession,
+    heftConfiguration: HeftConfiguration
+  ): Config.ReporterConfig {
+    const reporterOptions: IHeftJestReporterOptions = {
+      heftConfiguration,
+      debugMode: heftSession.debugMode
+    };
+
+    return [
+      path.resolve(__dirname, 'HeftJestReporter.js'),
+      (reporterOptions as unknown) as Record<string, unknown>
```

Comment:
You can avoid this cast by updating the `IHeftJestReporterOptions ` declaration to `interface IHeftJestReporterOptions extends Record<string, unknown>`

Reply:
I think the cast is the right thing.

I don't know that turning `HeftJestReporterOptions` into arbitrary property bag is worth skipping a one time cast.
We can improve the cast by skipping the `unknown`

Reply:
I agree with @hbo-iecheruo